starboard/raspi/shared/time_zone.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard/raspi/shared/time_zone.h"
+
+#include "starboard/extension/time_zone.h"
+
+#include <stdlib.h>
+#include <time.h>
+#include <cstring>
+
+#include "starboard/common/log.h"
+
+namespace starboard {
+namespace raspi {
+namespace shared {
+
+namespace {
+
+// Definitions of any functions included as components in the extension
+// are added here.
+
+bool SetTimeZone(const char* time_zone_name) {
```

Comment:
Can't we reuse the linux implementation? Potentially we can just add the relevant files to the gn BUILD file for raspi2.

Reply:
Sure, we can. Send this out just to test it works. One thing I'm not sure is where to put those shared file between Linux and Raspi2. Share we move the files from `starboard/linux/shared` to `starboard/shared/linux` because starboard/shared/linux/... is used in Raspi BUILD file? The directory is a little bit confusing.....

Reply:
Actually `starboard/linux/shared/` is used by Raspi as well. It's indeed confusing.....